2022-03-23 Updating Shiny dashboards from Excel via MongoDB (the Good, the Bad and the Shiny)/README.md

```diff
@@ -6,7 +6,25 @@
 Reads in data from a mongoDB collection. Includes a macro-enabled Excel workbook that can push data to a mongoDB collection at the push of a button (well 2 buttons...`CTRL+W`).
 
 This R package is based on the [`{mongo2shiny}`](https://github.com/MarkMc1089/mongo2shiny) template, which is
+<<<<<<< HEAD
+an extension of [`{nhsbsaShinyR}`](https://github.com/nhsbsa-data-analytics/nhsbsaShinyR) template developed by NHS Business Services Authority Data Analytics Learning Lab to use as a template for building NHSBSA branded R `{shiny}` dashboards.
+
+## Usage
+After initially cloning the repo, there will be no `data.rda` file in the `data` folder. This will cause an error in the case that the MongoDB connection is not available, as the fallback is to read the latest data. To ensure there is one, run the below code from the project root directory, making sure to source `R/data_prep.R` first.
+
+```
+library(dplyr)
+library(readxl)
+# Ensure you are in the project root folder
+sample_data <- read_excel("data-raw/nhs_pensions_FIC_data.example.xlsm")
+# You will need to source the data_prep.R file in the R folder
+data_prep(sample_data)
+```
+
+To run the app, just open `app.R` and click Run app in RStudio.
+=======
 an extension of [`{nhsbsaShinyR}`](https://github.com/nhsbsa-data-analytics/nhsbsaShinyR) template developed by NHS Business Services Authority Data Analytics Learning Lab to use as a template for building NHSBSA branded R `{shiny}` dashboards. 
+>>>>>>> 60adb8509e3444283a9124d8dbedbae4639eb756
```

Comment:
merge conflict is still in README. need to remove `<<<<<<<<<< HEAD`, and everything between `========` and `>>>>>>` inclusive.

Reply:
Done. That's what I get for editing files directly in GitHub!